scripts/world_bank/boundaries/country_boundaries_mcf_generator.py

```diff
@@ -0,0 +1,202 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Generates mcf files for country / AdministrativeArea0 boundaries from World Bank data.
+
+MCF files are generated for both high res data from the source, and simplified boundaries for
+interactive applications. Geojsons are simplified using
+https://geopandas.readthedocs.io/en/latest/docs/reference/api/geopandas.GeoSeries.simplify.html
+
+    Typical usage:
+    python3 country_boundaries_mcf_generator.py
+
+NOTE: this file generates temporary folders that are not deleted.
+"""
+import datacommons as dc
+import geopandas as gpd
+import glob
+import io
+import json
+import numpy as np
+import os
+import pandas as pd
+import requests
+import tempfile
+import zipfile
+
+from absl import app
+from absl import flags
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string(
+    'download_dir', '',
+    'Dir to use source data from. Leave blank to download from source.')
+flags.DEFINE_string(
+    'export_dir', '',
+    'Dir to output geojson coordinate files too. If blank, a temp folder will be used.'
+)
+flags.DEFINE_string(
+    'mcf_dir', '',
+    'Dir to output generated MCF files too. If blank, a temp folder will be used.'
+)
+
+DOWNLOAD_URI = 'https://development-data-hub-s3-public.s3.amazonaws.com/ddhfiles/779551/wb_boundaries_geojson_highres.zip'
+TMP_PATH = '/tmp/wb_boundaries_geojson_highres'
+GEO_JSON_PATH = 'WB_Boundaries_GeoJSON_highres/WB_countries_Admin0.geojson'
+ISO_A3_CODES_TO_IMPORT = ['COD', 'ROU', 'TLS', 'AND', 'IMN', 'UMI', 'USA']
```

Comment:
maybe a comment on what this extra map is for?

Reply:
good point, added